code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -156,8 +156,8 @@
 // Actions received from TGUI
 /mob/living/simple_animal/bot/secbot/ui_act(action, params)
 	. = ..()
-	if(. || (bot_cover_flags & BOT_COVER_LOCKED && !usr.has_unlimited_silicon_privilege))
-		return
+	if(.)
+		return TRUE
```

Comment:
In fact actually yeah, I'm not a fan of this.
What if someone wants to use ui_act for something, and they don't care about cover or silicon access? You're effectively silently blocking them from doing that

Reply:
But we want them to care about cover and silicon access, it's the whole thing about managing bots.